specs/WebResourceRequestedWorkers.md

```diff
@@ -0,0 +1,146 @@
+WebResourceRequested events for workers
+===
+
+# Background
+Currently [WebResourceRequested events](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/iwebview2webview5?view=webview2-0.8.355#add_webresourcerequested) are raised only for main page HTTP requests. We are extending [AddWebResourceRequestedFilter](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/iwebview2webview5?view=webview2-0.8.355#addwebresourcerequestedfilter) API to allow you additionally to subscribe to events raised from Service or Shared workers and fixing oop iframes events as part of the main page.
+
+# Examples
+Subscribe to WebResourceRequested event for service workers
+
+.Net
+```c#
+Controller.CoreWebView2.AddWebResourceRequestedFilter("*", CoreWebView2WebResourceContext.All, CoreWebView2WebResourceRequestSource.ServiceWorker);
+Controller.CoreWebView2.WebResourceRequested += (sender, e) =>
+{
+    string message =  "{" + WebResourceRequestedToJsonString(e.Request, e.RequestSource);
+    message += WebViewPropertiesToJsonString(sender) + "}";
```

Comment:
This looks like sort of logging. Do we have an example of a scenario an end developer wants to do with WebResourceRequested for service workers that we can demonstrate in the sample code instead?

Reply:
Updated sample with overriding image in request